pages/entity/_type/_.vue

```diff
@@ -141,6 +144,18 @@
         }));
       }
 
+      let contentfulClient;
+      if (query.mode === 'preview' && process.env['CTF_CPA_ACCESS_TOKEN']) {
+        contentfulClient = createClient(query.mode);
+      } else {
+        contentfulClient = createClient();
+      }
+      const setLocale = app.i18n.locale;
+      const isoLookUp = (code) => {
```

Comment:
As this duplicates code from pages/index.vue, I suggest creating a new plugin i18n.js which would accept the value of `app.i18n` as its parameter.